include/itkPhaseSymmetryImageFilter.h

```diff
@@ -81,26 +81,26 @@ class PhaseSymmetryImageFilter : public ImageToImageFilter<TInputImage,TOutputIm
   itkStaticConstMacro(InputImageDimension, unsigned int, TInputImage::ImageDimension);
   itkStaticConstMacro(OutputImageDimension, unsigned int, TOutputImage::ImageDimension);
 
-  /** Some convenient typedefs. */
-  typedef TInputImage                            InputImageType;
-  typedef typename    InputImageType::Pointer    InputImagePointer;
-  typedef typename    InputImageType::RegionType InputImageRegionType;
-  typedef typename    InputImageType::PixelType  InputImagePixelType;
+  /** Some convenient type alias. */
+  using InputImageType = TInputImage;
+  using InputImagePointer = typename    InputImageType::Pointer;
```

Comment:
Fix this and subsequent spaces?

Reply:
Thanks @dzenanz :eyes:. Done in 820024e.